prowler/lib/cli/parser.py

```diff
@@ -343,6 +343,15 @@ def __init_aws_parser__(self):
             default=None,
             help="Path for allowlist yaml file. See example prowler/config/allowlist.yaml for reference and format. It also accepts AWS DynamoDB Table ARN or S3 URI, see more in https://docs.prowler.cloud/en/latest/tutorials/allowlist/",
         )
+        # Allowlist
+        audit_tags_subparser = aws_parser.add_argument_group("Tags-based Scan")
+        audit_tags_subparser.add_argument(
+            "-t",
+            "--scan-tags",
+            nargs="+",
+            default=None,
+            help="Scan only resources with specific tags (Key=Value), e.g., Environment=dev Project=prowler",
+        )
```

Comment:
Please, include the parser test for these new arguments in `tests/lib/cli/parser_test.py`

Reply:
Done